app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -427,4 +436,21 @@ export default {
     }
   }
 }
+.action-button__wrap {
+  display: flex;
+  justify-content: flex-end;
+}
+.action-button {
+  position: fixed;
+  background: var(--white);
+  width: var(--space-two);
+  height: var(--space-two);
+  top: var(--space-mega);
+  z-index: var(--z-index-low);
+  margin-left: var(--space-half);
+  border-color: var(--color-border);
+  border-width: 0 0 0 1px;
+  border-style: solid;
+  border-radius: 50%;
```

Comment:
`--border-radius-rounded` instead of 50%

Reply:
Fixed https://github.com/chatwoot/chatwoot/pull/3507/commits/c1083a3d47d01058426949b6eb5bdba9c674846b